design/features/file-upload/README.md

```diff
@@ -0,0 +1,353 @@
+
+Form creators also noted that it will be important for the file upload questions to work well on mobile when filling in the form.
+
+Limit to 4 files per form submissions could be a blocker to some teams, if they needed more files they wouldn’t use GOV.UK Forms.
+
+It wasn't clear to the form creator what file formats are accepted and if this is communicated back to form fillers. 
+
+Form creators would like to have more control over what file types they accept.
+
+Some form creators indicated that form fillers should be able to change the file names after uploading, so that it can help with processing and make it more findable.
+
+Some users were concerned whether it would be safe to open the file attachments or receive files through email (with people’s personal data, like a passport photo).
+
+## Released feature
+
+### Form creator can add a file upload question
+!["What kind of answer do you need to this question" page. Screenshot](screenshots-released-version/000-create-file-upload-question-answer-type.png)
+
+*A page titled “What kind of answer do you need to this question” with a list of options to select from*
+
+When a user creates a question from this page they can select ‘File upload - people will only be able to upload on file’ as an option. We iterated the content multiple times to make it clearer that form fillers would be able to upload one file only. 
+
+**Error messages**
+
+When a form creator tried to add more than four file upload questions, we show an error message: “You cannot have more than 4 file upload questions in a form”
+
+### Form creator edits file upload question
+![Edit question page for file upload question.Screenshot](screenshots-released-version/001-edit-file-upload-question.png)
+
+*A page titled “Edit question”*
+
+The create and edit question page is tailored to be more specific to asking for a file. We removed the ‘Can answer more than once’ section as the file upload is restricted to one file per question.
+At the top of the page we communicate the restrictions for file upload and after feedback from users we included a copy about what files people can upload including the file sizes. The copy:
+
+> People can only upload one file for each file upload question. 
+>
+> A form can have up to 4 file upload questions.
+>
+> What files can people upload?
+> People can upload the following file types:
+> - CSV (.csv)
+> - image (.jpeg, .jpg, .png)
+> - Microsoft Excel Spreadsheet (.xlsx)
+> - Microsoft Word Document (.doc, .docx)
+> - PDF (.pdf)
+> - text (.json, .odt, .rtf, .txt)
+>
+> Each file must be smaller than 7MB.
+
+We used a specific copy to help form creators ask a file upload question:
```

Comment:
```suggestion
```

I don’t think this is needed. It’s mentioned at the top and we are about to explain the changes that were made below.